src/Builtin.ml

```diff
@@ -411,7 +411,7 @@ let prepare files =
         name, snd f @ extra
       with Not_found ->
         f
-  ) files @
```

Comment:
why don't you add steel_reference in hand_written, then? the behavior of the modules in hand_written is that they override anything otherwise extracted by F* (see line 405)